src/main/java/com/rultor/agents/req/StartsRequest.java

```diff
@@ -348,8 +348,11 @@ private DockerRun docker(final String type) throws IOException {
         if (nodes.isEmpty()) {
             throw new Profile.ConfigException(
                 String.format(
-                    "There is no '%s' section in .rultor.yml",
-                    type
+                    "There is no '%s' section in %s for branch %s in repo %s",
+                    type,
+                    ".rultor.yml",
+                    "master",
```

Comment:
@volodya-lombrozo you meant `profile.defaultBranch()` here? typo?

Reply:
@yegor256 Yes, forget about it. Fixed.